sdk/formrecognizer/ai-form-recognizer/tests.yml

```diff
@@ -20,3 +20,17 @@ stages:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+      CloudConfig:
+        Public:
+          SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
+          Location: "canadacentral"
```

Comment:
This will get overridden under the hood because of your global location parameter regardless, but I wonder if it would be a little clearer to use `${{ parameters.Location }}` here instead.